Sprint-3/alarmclock/alarmclock.js

```diff
@@ -1,4 +1,44 @@
-function setAlarm() {}
+function remainingTime(totalSeconds) {
+  let timeRemaining = document.getElementById("timeRemaining");
+  let minutes = Math.floor(totalSeconds / 60);
+  let seconds = totalSeconds % 60;
+
+   //timeRemaining.textContent = "Time Remaining  " + minutes + ":" + seconds;
+
+  if (minutes < 10) {
+    timeRemaining.textContent =
+      "Time Remaining  " + "0" + minutes + ":" + seconds;
+  } else if (seconds < 10) {
+    timeRemaining.textContent =
+      "Time Remaining  " + minutes + ":" + "0" + seconds;
+  } else {
+    timeRemaining.textContent = "Time Remaining  " + minutes + ":" + seconds;
+  }
+}
+
+
+function setAlarm() {
+  let inputTime = document.getElementById("alarmSet");
+  let totalSeconds = parseInt(inputTime.value);
+
+  if (isNaN(totalSeconds) || totalSeconds <= 0) {
```

Comment:
good use of different scenarios here to ensure it works as intended

Reply:
I will check that